C++ API parity: at::Tensor::output_nr

pbelevich opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #26332 C++ API parity: at::Tensor::requires_grad_
* #26251 C++ API parity: at::Tensor::detach
* #26217 C++ API parity: at::Tensor::version
* **#26216 C++ API parity: at::Tensor::output_nr**

Differential Revision: [D17427576](https://our.internmc.facebook.com/intern/diff/D17427576)

pbelevich committed:
C++ API parity: at::Tensor::output_nr

pbelevich committed:
Update on "C++ API parity: at::Tensor::output_nr"

pbelevich committed:
Update on "C++ API parity: at::Tensor::output_nr"

pbelevich committed:
Update on "C++ API parity: at::Tensor::output_nr"

pbelevich committed:
Update on "C++ API parity: at::Tensor::output_nr"

pbelevich committed:
Update on "C++ API parity: at::Tensor::output_nr"

pbelevich committed:
Update on "C++ API parity: at::Tensor::output_nr"

pbelevich committed:
Update on "C++ API parity: at::Tensor::output_nr"

pbelevich committed:
Update on "C++ API parity: at::Tensor::output_nr"

pbelevich committed:
Update on "C++ API parity: at::Tensor::output_nr"

pbelevich committed:
Update on "C++ API parity: at::Tensor::output_nr"

pbelevich committed:
Update on "C++ API parity: at::Tensor::output_nr"

pbelevich committed:
Update on "C++ API parity: at::Tensor::output_nr"

yf225 reviewed (approved):
Thanks a lot @pbelevich !

pbelevich commented:
@pytorchbot retest this please

pbelevich committed:
Update on "C++ API parity: at::Tensor::output_nr"


Differential Revision: [D17427576](https://our.internmc.facebook.com/intern/diff/D17427576)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: at::Tensor::output_nr"


Differential Revision: [D17427576](https://our.internmc.facebook.com/intern/diff/D17427576)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: at::Tensor::output_nr"


Differential Revision: [D17427576](https://our.internmc.facebook.com/intern/diff/D17427576)

[ghstack-poisoned]
